tensorflow_addons/activations/rrelu.py

```diff
@@ -26,6 +31,7 @@ def rrelu(
     upper: Number = 0.3333333333333333,
     training: Optional[bool] = None,
     seed: Optional[int] = None,
+    gs: Optional[tf.random.experimental.Generator] = None,
```

Comment:
Do we have a policy about the use of experimental features in TensorFlow? What are the conditions in which we should accept the usage?

For example, I'd be ok if we use experimental features in tests, but not really in user-facing code. That's because it's ok if the build breaks because of an experimental feature, but it's not ok if the code the users are using is breaking.

@fsx950223  Maybe we can wait until the Generator is not experimental anymore?

Reply:
Yes, I will wait until stable API is released

Reply:
https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/random/Generator

Reply:
Stable API is available on TF2.2 which is what master builds against. Please merge master into this PR and should be good to update